[2020-06-02 21:20:31,093] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_simple_dag.load 2020-06-02T09:18:00+00:00 [queued]>
[2020-06-02 21:20:31,101] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_simple_dag.load 2020-06-02T09:18:00+00:00 [queued]>
[2020-06-02 21:20:31,102] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-02 21:20:31,102] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-02 21:20:31,102] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-02 21:20:31,128] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): load> on 2020-06-02T09:18:00+00:00
[2020-06-02 21:20:31,133] {standard_task_runner.py:53} INFO - Started process 29925 to run task
[2020-06-02 21:20:31,228] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_simple_dag.load 2020-06-02T09:18:00+00:00 [running]> nineleaps-ThinkPad-E470
[2020-06-02 21:20:31,248] {logging_mixin.py:112} INFO - 2020-06-02
[2020-06-02 21:20:31,248] {taskinstance.py:1145} ERROR - an integer is required (got type str)
Traceback (most recent call last):
  File "/home/nineleaps/Desktop/work_from_home_assignments/python_assignment/asf/venv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/nineleaps/Desktop/work_from_home_assignments/python_assignment/asf/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/nineleaps/Desktop/work_from_home_assignments/python_assignment/asf/venv/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/Desktop/work_from_home_assignments/python_assignment/asf/airflow_home/pwd/dags/my_simple_dag.py", line 99, in load_data_func
    output_rows=load_data(report_name)
  File "/home/nineleaps/Desktop/work_from_home_assignments/python_assignment/asf/airflow_home/pwd/dags/my_simple_dag.py", line 73, in load_data
    date=int(file_date.replace('-',''))
TypeError: an integer is required (got type str)
[2020-06-02 21:20:31,249] {taskinstance.py:1202} INFO - Marking task as FAILED.dag_id=my_simple_dag, task_id=load, execution_date=20200602T091800, start_date=20200602T155031, end_date=20200602T155031
[2020-06-02 21:20:41,053] {logging_mixin.py:112} INFO - [2020-06-02 21:20:41,052] {local_task_job.py:103} INFO - Task exited with return code 1
